Tools/autotest/carbonix.py

```diff
@@ -60,6 +60,8 @@ def TestESCTelemetry(index):
             index = int(index)
             self.context_push()
             self.context_collect('STATUSTEXT')
+            # disable ArduPilot's engine prearm check to use wait_ready_to_arm
+            self.set_parameter('BIT_PREARM_DIS', 0b100)
```

Comment:
Let's disable the engine prearm check this bit within the top-level test, instead of within the individual tests, so that we just do this once.